docker/docker-compose-tests.yml

```diff
@@ -0,0 +1,38 @@
+version: '2.2'
+services:
+  rally:
+    image: elastic/rally:${RALLY_VERSION}
+    container_name: rally
+    command: ${TEST_COMMAND}
```

Comment:
I assume this is blocking and we won't run into a race condition in our tests where we basically do a `docker-compose up` followed by a `docker-compose down`?

Reply:
Correct, docker-compose will return control only when all services are up and satisfy the declared dependencies.